cypress/integration/pages/mostReadPage/tests.js

```diff
@@ -0,0 +1,41 @@
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`No testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {
+    describe(`Tests for ${service} ${pageType}`, () => {
+      it('should have a visually hidden top-level header', () => {
```

Comment:
Does this page have a visually hidden header?

Reply:
not that I know of... I'll update the description ✍️

Reply:
may be this test could be modified to fit in more like a e2e test ie) clicking on a page takes to an existing page and on doing a back the most read component is displayed on the page?